.circleci/config.yml

```
@@ -5,27 +5,25 @@ jobs:
     docker:
       - image: node:10.16.0
         environment:
-          NODE_ENV: development
-          MONGO_URI: mongodb://localhost:27017/crowi
           REDIS_URL: redis://localhost:6379/crowi
-          ELASTICSEARCH_URI: //localhost:9200/
           PLANTUML_URI: //localhost:8080
           FILE_UPLOAD: local
           MONGOMS_DOWNLOAD_MIRROR: https://downloads.mongodb.org
-      - image: mongo:3.6.13
+          MONGOMS_DISABLE_POSTINSTALL: 1
+      - image: circleci/redis:5
```

```suggestion
      - image: circleci/redis:5.0.5
```

I would like to specify a minor version to change the version explicitly.